src/plugins/mf_priority.cpp

```diff
@@ -184,6 +204,71 @@ static int priority_cb (flux_plugin_t *p,
         return -1;
     }
 
+    b = static_cast<bank_info *> (flux_jobtap_job_aux_get (
+                                                    p,
+                                                    FLUX_JOBTAP_CURRENT_JOB,
+                                                    "mf_priority:bank_info"));
+
+    if (b == NULL) {
+        flux_jobtap_raise_exception (p, FLUX_JOBTAP_CURRENT_JOB, "mf_priority",
+                                     0, "job.state.priority: bank info is " \
+                                     "missing");
+
+        return -1;
+    }
+
+    std::map<int, std::map<std::string, struct bank_info>>::iterator it;
+    std::map<std::string, struct bank_info>::iterator bank_it;
+
+    if (b->max_run_jobs == BANK_INFO_MISSING) {
+        // try to look up user again
+        it = users.find (userid);
+        if (it == users.end ()) {
+            return flux_jobtap_priority_unavail (p, args);
+        } else {
+            // make sure user belongs to bank they specified; if no bank was
+            // passed in, look up their default bank
+            if (bank != NULL) {
+                bank_it = it->second.find (std::string (bank));
+                if (bank_it == it->second.end ())
+                    flux_jobtap_raise_exception (p, FLUX_JOBTAP_CURRENT_JOB,
+                                                 "mf_priority",
+                                                 0,
+                                                 "job.state.priority: user " \
+                                                 "does not belong to " \
+                                                 "specified bank");
```

Comment:
The error emitted by `flux_jobtap_raise_exception()` is directed at the user. It may be confusing to include the plugin topic string in the exception note, so I'd just drop that. Also, it may be useful to include the specified bank name, e.g. (untested):

```suggestion
                    flux_jobtap_raise_exception (p, FLUX_JOBTAP_CURRENT_JOB,
                                                 "mf_priority",
                                                 0,
                                                 "not a member of bank %s",
                                                 bank);
```

Reply:
Thanks for this suggestion! I just squashed and pushed a change onto [d454f3f](https://github.com/flux-framework/flux-accounting/pull/227/commits/d454f3f16b64293a80ab0b3013d76ec360e28fc3) that includes the bank name in the exception message. I also added a test in `t1012-mf-priority-load.t` that checks for this exception message when a user submits a job using an incorrect bank name before the user/bank information is sent to the plugin.